k8s/pkg/apis/longhorn/v1beta2/backup.go

```diff
@@ -7,6 +7,7 @@ type BackupState string
 const (
 	BackupStateNew        = BackupState("")
 	BackupStatePending    = BackupState("Pending")
+	BackupStateStarting   = BackupState("Starting")
```

Comment:
This is not necessary, since we did not introduce this state from backup status anymore.

Reply:
Removed. Thank you.